infra/spawn_data_pipeline.py

```diff
@@ -0,0 +1,111 @@
+#!/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Mozilla
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script triggers the data pipeline for the bugbug project
+"""
+
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+import jsone
+import requests.packages.urllib3
+import taskcluster
+import yaml
+
+requests.packages.urllib3.disable_warnings()
+
+TASKCLUSTER_DEFAULT_URL = "https://taskcluster.net"
+
+
+def get_taskcluster_options():
+    """
+    Helper to get the Taskcluster setup options
+    according to current environment (local or Taskcluster)
+    """
+    options = taskcluster.optionsFromEnvironment()
+    proxy_url = os.environ.get("TASKCLUSTER_PROXY_URL")
+
+    if proxy_url is not None:
+        # Always use proxy url when available
+        options["rootUrl"] = proxy_url
+
+    if "rootUrl" not in options:
+        # Always have a value in root url
+        options["rootUrl"] = TASKCLUSTER_DEFAULT_URL
+
+    return options
+
+
+def main():
```

Comment:
@marco-c This is basically https://github.com/mozilla/task-boot/issues/14 without adding a json-e function.